charts/sftd/values.yaml

```diff
@@ -108,6 +108,10 @@ multiSFT:
   secret:
   turnServerURI:
 
+# This is the secret used for authentication when the multiSFT feature is disabled
+# Do not use this when multiSFT is enabled, and instead use the `multiSFT.secret` field
+sftTokenSecret:
```

Comment:
can you put a commented out example secret here, so someone filling this out knows what the secret should look like, be escaped, quoted, etc?